pages/common/git-sizer.md

```diff
@@ -0,0 +1,16 @@
+# git sizer
```

Comment:
Are you sure this is spelt correctly? I get a `sizer` is not a git command.

Reply:
If it's in the path, it should work with a space I think. Otherwise `git-sizer` should be used.

Looks like it is spelt correctly though.

Reply:
You can invoke it by running either `git-sizer` or `git sizer`. I've done it with both.

Reply:
Yep, this is fine.

Reply:
Huh. It must be an extension or bundled as part of a newer version of git then.